WalletWasabi/Extensions/StringExtensions.cs

```diff
@@ -60,5 +60,20 @@ public static string TrimEnd(this string me, string trimString, StringComparison
 			}
 			return me;
 		}
+
+		public static bool IsTrimable(this string? me)
```

Comment:
Looks good, I would do it like bool TryTrim(this string? me, out string trimmed). So you don't need to Trim again. When evaluating just omit the output with out _

Reply:
If we do it like this, the `PasswordHelper.IsTrimable()` would not turn unnecessary? Because the TryTrim would do the exact same thing like `PasswordHelper.IsTrimable()` before this PR.

Reply:
What if the `IsTrimable()` extension method just checks if the first or last char is whitespace or not?

Reply:
I would like to avoid that as the Trim method could be more complex inside - I am not sure.

Reply:
I did research before I suggest that. The `Trim()` method does nothing but finds the first non whitespace char from the beginning and the end of the string and returns a substring.

So logically if the first or last char is whitespace then the string trimmable.

![image](https://user-images.githubusercontent.com/16364053/98261580-e4d15400-1f84-11eb-928b-ea9231d9eb90.png)
![image](https://user-images.githubusercontent.com/16364053/98261504-d3884780-1f84-11eb-8eaf-8788c73f81aa.png)